src/views/ManageTeamHealth.vue

```diff
@@ -224,8 +256,26 @@ export default {
               class="health-check__row"
             >
               <td class="tooltip-target health-check__criteria">
-                <i class="fa icon icon--criteria" :class="criteria.icon"/>
-                <span>{{ criteria.label }}</span>
+                <v-popover offset="16">
+                  <!-- This will be the popover target (for the events and position) -->
+                  <i class="fa icon icon--criteria" :class="criteria.icon" />
+                  <span>{{ criteria.label }}</span>
+
+                  <!-- Popover content -->
+                  <template slot="popover" class="popover__criteria">
```

Comment:
Are we following the rule of if there is more than one attribute, they have their own lines?

Reply:
We're using `prettier` which is an automatic code formatting package. I've not configured it outside of the default options, so we can refine it going forward. I think the default is it will put them all on their own lines when there is more than 3 attributes

Reply:
Ahh fair enough 👍